erika/erika.py

```diff
@@ -10,75 +10,82 @@ class Erika:
     conversion_table_path = "erika/charTranslation.json"
```

Comment:
Great for now :) 

I would have preferred a stepdown-rule function ordering, though, so called functions are lower in the file than calling ones.  
This would improve readability.

Reply:
Will refactor after merge